README.md

```diff
@@ -1,15 +1,15 @@
 # Devtools
 
-[![UnitTests](https://github.com/Lightning-AI/devtools/actions/workflows/ci-testing.yml/badge.svg?event=push)](https://github.com/Lightning-AI/devtools/actions/workflows/ci-testing.yml)
-[![Apply checks](https://github.com/Lightning-AI/devtools/actions/workflows/ci-use-checks.yml/badge.svg?event=push)](https://github.com/Lightning-AI/devtools/actions/workflows/ci-use-checks.yml)
-[![Documentation Status](https://readthedocs.org/projects/lightning-devtools/badge/?version=latest)](https://lightning-devtools.readthedocs.io/en/latest/?badge=latest)
-[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/Lightning-AI/devtools/main.svg?badge_token=mqheL1-cTn-280Vx4cJUdg)](https://results.pre-commit.ci/latest/github/Lightning-AI/devtools/main?badge_token=mqheL1-cTn-280Vx4cJUdg)
+[![UnitTests](https://github.com/Lightning-AI/lightning-tools/actions/workflows/ci-testing.yml/badge.svg?event=push)](https://github.com/Lightning-AI/lightning-tools/actions/workflows/ci-testing.yml)
```

Comment:
The repo hasn't been renamed. I assumed this was the desired name

Reply:
yes, lets keep the repo name
```suggestion
[![UnitTests](https://github.com/Lightning-AI/devtools/actions/workflows/ci-testing.yml/badge.svg?event=push)](https://github.com/Lightning-AI/devtools/actions/workflows/ci-testing.yml)
```

Reply:
Done in https://github.com/Lightning-AI/utilities/pull/29